include/clap/ext/audio-ports.h

```diff
@@ -65,15 +65,14 @@ typedef struct clap_audio_port_info {
    clap_id in_place_pair;
 } clap_audio_port_info_t;
 
-// The audio ports scan has to be done while the plugin is deactivated.
 typedef struct clap_plugin_audio_ports {
    // Number of ports, for either input or output
-   // [main-thread]
+   // [main-thread & !active]
```

Comment:
The ports layout has to be known before the plugin is activated, but then why would you forbid reading the ports?